wordssite/dictionary/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+from .models import Word,Translation
+from django.core.exceptions import ValidationError
+ 
+ 
+class WordForm(forms.ModelForm):
+    def clean_word(self):
+        word = self.cleaned_data["word"].title()
+
+        if Word.objects.filter(word = word).count()>0:
+            raise ValidationError(f"{word} already exist")
+
+        if any(str.isdigit(i) for i in word):
```

Comment:
how about special chars like !#$?} ?:):/;) 
Maybe we should only check if letters are used, smth like:
https://stackoverflow.com/questions/18667410/how-can-i-check-if-a-string-only-contains-letters-in-python